package.json

```diff
@@ -32,24 +32,24 @@
     "@babel/plugin-transform-modules-commonjs": "^7.12.1",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-typescript": "^7.12.7",
-    "@ceramicnetwork/cli": "^1.0.0-rc.13",
-    "@ceramicnetwork/common": "^1.0.0-rc.6",
-    "@ceramicnetwork/core": "^1.0.0-rc.12",
-    "@ceramicnetwork/streamid": "^1.0.0-rc.3",
-    "@ceramicnetwork/stream-tile": "^1.0.0-rc.6",
-    "@ceramicnetwork/http-client": "^1.0.0-rc.6",
-    "@ceramicnetwork/ipfs-daemon": "^1.0.0-rc.9",
+    "@ceramicnetwork/cli": "^1.1.0-rc",
+    "@ceramicnetwork/common": "^1.1.0-rc",
+    "@ceramicnetwork/core": "^1.2.0-rc",
+    "@ceramicnetwork/http-client": "^1.0.7-rc",
```

Comment:
can (need to?) drop the `-rc` from the end

Reply:
@stbrody drop the rc from all?

Reply:
yeah.  TBH I didn't even know you *could* just leave the `-rc` suffix without specifying a specific RC version.  I wonder what it even means/does?  Will it just take any RC for version 1.0.7?  In any case would be good to drop and require the actual version 1.0.7 or later, not an RC for it.